site/pages/react-native/getting-started/getting-started-expo.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Getting started with Account Kit on Expo
+description: A guide on integrating Account Kit within a React Native Expo application
+---
+
+:::warning
+To use Account Kit in your React Native application, ensure you are on React Native version 0.74 or higher.
+:::
+
+## Getting Started
+
+We would go through the steps to get your environment setup for using Account Kit within a React Native application on Expo.
+
+### Upgrade to the latest version of Expo
+
+The first thing we need to do is make sure we're on the latest version of Expo (SDK 52 or later). The reason for this is that we need React Native version 0.74 or higher because it has `TextEncoder` natively supported.
```

Comment:
Is this the right version of expo? should we be on a version of expo that's using 0.76?

Reply:
Gotcha, we should suggest 0.76 here.